### PR TITLE
Change throwError to throwOnError and add throwOnWarning

### LINKED ISSUE #16: Change `throwError` to `throwOnError` and `throwOnWarning`
@TrySound  would accept a PR to these changes?

The reason is add the ability to not break on warnings for example.

### PR DESCRIPTION
**Pull request by jonataswalker**
### Breaking changes:
  - change `throwError` to `throwOnError`

### Add `throwOnWarning`

Fixes #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,30 +34,37 @@ rollup({
 
 See more options here [eslint-config].
 
-### throwError
+### throwOnError
 
-Type: `boolean`  
+Type: `boolean`
```
Comment (TrySound) [inline README.md]: Bring back double space please. It's for new line break.

**Comment by jonataswalker** [comment]
Not sure about this failing error:

```
Error: Module /home/travis/build/TrySound/rollup-plugin-eslint/node_modules/minimatch/minimatch.js does not export Minimatch (imported by /home/travis/build/TrySound/rollup-plugin-eslint/node_modules/rollup-pluginutils/dist/pluginutils.es6.js)
```
Tried to update `npm` but nothing changed. It doesn't occur on my machine.

**Comment by jonataswalker** [comment]
@TrySound help :)

**Comment by TrySound** [comment]
Landed here https://github.com/TrySound/rollup-plugin-eslint/releases/tag/v4.0.0
Thanks for your work

**Pull request closed without merging**